src/HoneybadgerServiceProvider.php

```diff
@@ -125,6 +125,36 @@ private function registerEventHooks()
                 }
             });
         });
+
+        /** @param string|array|null $environments */
+        Event::macro('thenPingHoneybadgerFromConfig', function (string $configKey, $environments = null) {
+            $id = config($configKey);
```

Comment:
Shouldn't you read config from honeybadger config? As in `config("honeybadger.checkins.$configKey")`

Reply:
I left it up to the consumer as to where they store the configuration - if they want to pull it from the honeybadger config they can, or if they want to have it in an app-specific config file (so that package upgrades that modify the honeybadger config file don't potentially cause weird merge issues).